src/app/user/user.service.ts

```diff
@@ -0,0 +1,16 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class  UserService {
+
+  constructor(private httpClient: HttpClient) { }
+
+  public user(){
+    return this.httpClient.get(`https://secure-refuge-14993.herokuapp.com/list_users`);
+ 
+  }
+
+}
```

Comment:
each and every file should be formatted from now on.
this will not be ACCEPTABLE.
no extra lines should be there
Ctrl shift I se format karo install prettier